packages/client/tailwind.config.js

```diff
@@ -20,6 +20,22 @@ export default {
         md: ['1rem', '1.5rem'],
       },
     },
+    screens: {
+      sm: '640px',
+      // => @media (min-width: 640px) { ... }
+
+      md: '768px',
+      // => @media (min-width: 768px) { ... }
+
+      lg: '1024px',
+      // => @media (min-width: 1024px) { ... }
+
+      xl: '1280px',
+      // => @media (min-width: 1280px) { ... }
+
+      '2xl': '1536px',
+      // => @media (min-width: 1536px) { ... }
+    },
```

Comment:
No need to overwrite the defaults, its fine to comment the values for referance purposes, I propose we have only add to these

 ```suggestion
      screens: {
        minimum: '320px',
        xs: '430px',
        // sm 640px
        // md 768px
        // lg 1024px
        // xl 1280px
        // 2xl 1536px
      },
```